programs/rewards/tests/rewards/utils.rs

```diff
@@ -250,19 +251,36 @@ impl TestRewards {
                 &self.reward_pool.pubkey(),
                 &self.token_mint_pubkey,
                 &self.vault_pubkey,
-                &self.fill_authority.pubkey(),
+                &fill_authority.pubkey(),
                 from,
                 amount,
                 distribution_ends_at,
             )],
             Some(&context.payer.pubkey()),
-            &[&context.payer, &self.fill_authority],
+            &[&context.payer, fill_authority],
             context.last_blockhash,
         );
 
         context.banks_client.process_transaction(tx).await
     }
 
+    pub async fn fill_vault(
```

Comment:
to my mind this method is redundant, because the method it calls do the same, it's just another redirection level